src/config/rt.rs

```diff
@@ -218,6 +218,10 @@ pub struct RtcServe {
     pub proxies: Option<Vec<ConfigOptsProxy>>,
     /// Whether to disable auto-reload of the web page when a build completes.
     pub no_autoreload: bool,
+    /// Path to crt file.
+    pub crt: PathBuf,
```

Comment:
Use `Option<PathBuf>` or `OsStr` for these types

Reply:
Wrapped key/crt member with Option.